From c07bca18ccd64f095ddb4e21922b642fafad4507 Mon Sep 17 00:00:00 2001
From: Dom Cobley <popcornmix@gmail.com>
Date: Fri, 31 Jul 2026 19:27:09 +0100
Subject: [PATCH 03/18] avcodec/h264_ps: fix swapped MVC profile comments

profile_idc 118 is Multiview High and 128 is Stereo High, cf. H.264 Annex A
and AV_PROFILE_H264_MULTIVIEW_HIGH / AV_PROFILE_H264_STEREO_HIGH in defs.h.
The comments had the two the wrong way round.

Signed-off-by: Dom Cobley <popcornmix@gmail.com>
---
 libavcodec/h264_ps.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libavcodec/h264_ps.c b/libavcodec/h264_ps.c
index e72d39ea8d0..d59fd2667e3 100644
--- a/libavcodec/h264_ps.c
+++ b/libavcodec/h264_ps.c
@@ -340,8 +340,8 @@ int ff_h264_decode_seq_parameter_set(GetBitContext *gb, AVCodecContext *avctx,
         sps->profile_idc ==  44 ||  // Cavlc444 profile
         sps->profile_idc ==  83 ||  // Scalable Constrained High profile (SVC)
         sps->profile_idc ==  86 ||  // Scalable High Intra profile (SVC)
-        sps->profile_idc == 118 ||  // Stereo High profile (MVC)
-        sps->profile_idc == 128 ||  // Multiview High profile (MVC)
+        sps->profile_idc == 118 ||  // Multiview High profile (MVC)
+        sps->profile_idc == 128 ||  // Stereo High profile (MVC)
         sps->profile_idc == 138 ||  // Multiview Depth High profile (MVCD)
         sps->profile_idc == 144) {  // old High444 profile
         sps->chroma_format_idc = get_ue_golomb_31(gb);
-- 
2.53.0

